### ISSUE TITLE
runoak diff erroneously names "relationships" --> "mappings" in markdown summary

### ISSUE THREAD

**Issue by matentzn** [INCATools/ontology-access-kit#735]
The "Relation ship added" section is erroneously called "Mappings added".

Example: https://github.com/obophenotype/human-phenotype-ontology/pull/10446#issuecomment-2041475692

```
reports/difference_release_base.md: tmp/hp-released.obo $(KGCL_ONTOLOGY)
	runoak -i simpleobo:tmp/hp-released.obo diff -X simpleobo:$(KGCL_ONTOLOGY) -o $@ --output-type md
```

**Comment by hrshdhgd** [comment]
So there might be some misunderstanding on my part or. bug in code. The table created under `Mappings added` are derived from the `Change` object `EdgeCreation`. This seems to be applicable for both relationship and mapping. How do we distinguish the two @cmungall ?

Similarly
 - Mappings removed: EdgeDeletion
 - Mappings changed: EdgeChange

**Comment by cmungall** [comment]
edge is very generic in KGCL, it encompasses relationships and mappings

**Comment by cmungall** [comment]
OK, I misspoke. edges is KGCL are the same as edges in OAK

this code is not quite right:

```python
            if mappings_added_set:
                for mapping in mappings_added_set:
                    predicate, xref = mapping
                    edge_created = EdgeCreation(
                        id=_gen_id(),
                        subject=entity,
                        predicate=predicate,
                        object=xref,
                    )
                    yield edge_created
            if mappings_removed_set:
                for mapping in mappings_removed_set:
                    predicate, xref = mapping
                    deleted_edge = EdgeDeletion(
                        id=_gen_id(),
                        subject=entity,
                        predicate=predicate,
                        object=xref,
                    )
                    yield deleted_edge
            if mapping_changed_set:
                for changes in mapping_changed_set:
                    object, new_predicate, old_predicate = changes
                    edge_change = EdgeChange(
                        id=_gen_id(),
                        about_edge=Edge(subject=entity, predicate=old_predicate, object=object),
                        old_value=old_predicate,
                        new_value=new_predicate,
                    )

                    yield edge_change
```

simply change `edge_` to `mapping_` and `Edge*` to `Mapping*` to get the correct KGCL types

**Comment by hrshdhgd** [comment]
Fixed by #775
